logger/src/metrics.rs

```diff
@@ -186,6 +186,10 @@ pub struct PatchRequestsMetrics {
     pub network_count: SharedMetric,
     /// Number of failures in PATCHing a net device.
     pub network_fails: SharedMetric,
+    /// Number of PATCHs for configuring the machine.
+    pub machine_cfg_count: SharedMetric,
```

Comment:
I'm not sure this is such an useful metric. Our number of metrics keeps increasing and I think we should start being selective on what is truly useful and what is just fluff.
@firecracker-microvm/compute-capsule

Reply:
I think it's useful. We'd want to see the success rate of each operation. With just the failure metric, we don't have the full picture of how frequent failures are.

Reply:
I totally agree for things that regularly fail and fail quietly.

This is an error case that blocks the user from configuring the microVM, I believe increments on this error is **absolute zero** in production.

If the API call fails one of two things happens:

1. That microVM is failed because of bad user-config (which the user is notified of through the API response) and the microVM is killed or just left around doing nothing -> **metrics are not being published**.
1. The API call is retried by the user, this time with the proper config, microVM is started, metrics are published and we have this useless metric (and others like it) incremented that brings no value as far as I can see.